epoxy-annotations/build.gradle

```diff
@@ -1,7 +1,9 @@
 apply plugin: 'java'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+apply from: "$rootDir/blessedDeps.gradle"
+
+sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
should we make a `deps.JAVA_VERSION` reference instead so we can just define the version to be `JavaVersion.VERSION_1_7` in the deps file?